ahrefs/monorobot
# delete labels in the attachment messege

## Description of the task

Deleted labels in the attachment of a notification as label information is already included in the PR/issue names.

## How to test

Run the following command with a json file related to PR / issue. One should expect that the message generated no longer contains labels in the attachment.

```
./notabot check --json mock_payloads/SOME_FILE
```

## References

- Slack discussion:
https://ahrefs.slack.com/archives/CKZANG2TE/p1591094809057500
https://ahrefs.slack.com/archives/CKZANG2TE/p1591122595058600
